ntc_templates/templates/fsas_sir_show_ether.textfsm

```diff
@@ -0,0 +1,30 @@
+Value Required ETHER_GROUP (\d+)
+Value Required ETHER_PORT (\d+)
+Value DESCRIPTION (\S+)
+Value STATUS (.+?)
+Value STATUS_AUTO (auto)
+Value STATUS_SPEED ([\d\.]+[MG])
```

Comment:
I wasn't able to find an instance where a literal period shows up in STATUS_SPEED.
If you're aware of an instance of this, please add a copy of that raw cli output to this PR. Thank you!

If you accept this change, please click the "commit suggestions" button.

```suggestion
Value STATUS_SPEED ([\d]+[MG])
```